src/Sulu/Bundle/AdminBundle/Resources/js/components/InfiniteScroller/README.md

```diff
@@ -0,0 +1,22 @@
+The 'InfiniteScroller' component enables a pagination where the user will need to scroll to the bottom of the page to 
+receive further content. The component finds its scrollable container automatically by traversing the DOM-Tree upwards
+until it detects a container with the styling `overflow: scroll` or `overflow: auto` applied.
+
+```javascript static
+const handleLoad = (pageToLoad) => {
+    // do something
```

Comment:
Would it be hard to add some kind of `alert` or whatever in here? Otherwise it is hard to see if it is working...

Reply:
Ah, just saw that it is `static`... Is that necessary?